### ISSUE TITLE
Scrollview not driving overlay after you scrolled down and scrolled to top.

### ISSUE THREAD

**Issue by Said13** [applidium/OverlayContainer#43]
Scrollview not driving overlay after you scrolled down and scrolled to top.

**Comment by gaetanzanella** [comment]
It looks fine in the example. Please provide something to prove it.

**Comment by Said13** [comment]
Yeah, in the example it works fine. I recorded video, could you give me your email so I can send it to you?

**Comment by gaetanzanella** [comment]
You can add the video to the issue. But here is my email gaetan.zanella@fabernovel.com.

**Comment by Said13** [comment]
Fixed it by setting on

> bounce on scroll

 and 

> bounce vertically

**Comment by gaetanzanella** [comment]
Good to know ! 👍

**Comment by pushpendra996** [comment]
> Fixed it by setting on `bounce on scroll` and `bounce vertically ` scroll view properties

Thanks, It's worked for me also!
